keps/sig-storage/20190122-csi-inline-volumes.md

```diff
@@ -134,27 +132,44 @@ A driver may be able to support either PV/PVC-originated or pod spec originated
 ### Inline CSI lifecycle setting
 A CSI driver will have to indicate the inline volume lifecycle, that it supports, as either `persistent` or `ephemeral`.  This setting will be stored in a [`CSIDriver` configuration CRD](https://github.com/kubernetes/enhancements/issues/594) and will be checked at runtime by Kubelet and external CSI components for enforcement.  The setting is to inform the Kubelet and other CSI components how to process inline volume information for proper setup.
 
-#### Persistent inline mode
-For CSI drivers that indicate they support persistent inline volumes, the `CSIDriver.lifecycle` value should be set to `persistent` (default of if not provided).  When a driver is used to handle volumes from an inline context: 
-* Volumes are assumed to be originated from an inline pod spec.
+### Persistent inline mode
+For CSI drivers that indicate they support persistent inline volume definitions, the `CSIDriver.lifecycle` value should be set to `persistent` (default if not provided).  When a persistent inline driver is used to handle volume operations: 
```

Comment:
is the `lifecycle` field new and being introduced as part of this? if so, follow the [conventions for enum values](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#constants): `Persistent` and `Ephemeral`

Reply:
Yes, lifecycle will be new. I will follow up on conventions.  Thanks.